packages/datadog-instrumentations/src/oracledb.js

```diff
@@ -21,7 +21,7 @@ function finish (err) {
   finishChannel.publish(undefined)
 }
 
-addHook({ name: 'oracledb', versions: ['5'] }, oracledb => {
+addHook({ name: 'oracledb', versions: ['>=5'] }, oracledb => {
```

Comment:
Does this single line also ensure we are _testing_ version 6, or we are just enabling tracing when version is 6 without testing it? In the latter case we should add a test.

Reply:
it is tested: https://github.com/DataDog/dd-trace-js/actions/runs/8384222443/job/22961264383?pr=4182 
```
with oracledb >=5 (6.4.0)
        without configuration
          with connection
            ✔ should be instrumented for promise API
            ✔ should restore the parent context in the promise callback
            ✔ should be instrumented for callback API
            ✔ should restore the parent context in the call
            ...
```

Reply:
Test versions are inferred from the version range reported by the integration by default, so whenever we change the range the tests automatically reflect that as well.

Reply:
All clear, thank you